sw/host/provisioning/ft_lib/src/lib.rs

```diff
@@ -423,9 +443,14 @@ fn provision_certificates(
     }
 
     // Validate the certificate endorsements with OpenSSL.
-    if !host_endorsed_certs.is_empty() {
-        validate_certs_chain(ca_certificate.to_str().unwrap(), &host_endorsed_certs)?;
+    log::info!("Validating DICE certificate chain with OpenSSL ...");
+    validate_cert_chain(ca_certificate.to_str().unwrap(), &dice_cert_chain, true)?;
+    log::info!("Success.");
+    log::info!("Validating SKU-specific certificates with OpenSSL ...");
+    if !sku_specific_certs.is_empty() {
+        validate_certs(ca_certificate.to_str().unwrap(), &sku_specific_certs)?;
```

Comment:
why do you need both `validate_certs()` and `validate_cert_chain()` 

You could create a vector of two certs here, replacing the second element with one SKU cert at a time and invoke `validate_certs()` on this vector in the loop

Reply:
good point. Fixed.